src/dataset-builder/DatasetBuilderDetails.ts

```diff
@@ -62,37 +62,36 @@ const TileDisplay = (props: DomainDisplayProps) => {
 };
 
 interface DatasetBuilderDetailsProps {
-  datasetId: string;
+  snapshotId: string;
 }
 
-export const DatasetBuilderDetails = ({ datasetId }: DatasetBuilderDetailsProps) => {
-  const [datasetDetails, loadDatasetDetails] = useLoadedData<DatasetModel>();
-  const [datasetRoles, loadDatasetRoles] = useLoadedData<string[]>();
+export const DatasetBuilderDetails = ({ snapshotId }: DatasetBuilderDetailsProps) => {
+  const [snapshotDetails, loadSnapshotDetails] = useLoadedData<Snapshot>();
+  const [snapshotBuilderSettings, loadSnapshotBuilderSettings] = useLoadedData<SnapshotBuilderSettings>();
+  const [datasetRoles, loadSnapshotRoles] = useLoadedData<string[]>();
```

Comment:
Roles are also inaccessible, we may need to infer the role from the response we get back from `getSnapshotBuilderSettings`

Reply:
The user should still be able to load their snapshot roles.